backend/app/controllers/MenteesController.php

```diff
@@ -38,6 +42,101 @@ public function singleMentee($id) {
         ]);
     }
 
+    public function login()
+    {
+
+        $credentials = request()->get(['email', 'password']);
+
+        if(!$credentials['email'] || !$credentials['password']) {
+            response()->exit([
+                'error' => '404',
+                'message' => "One of required fields isn't filled.",
+            ]);
+        }
+        $user = User::where('email', $credentials['email'])->first();
+
+        $user = auth()->login($credentials);
+
+        if (!$user) {
+            response()->exit(auth()->errors());
+        }
+
+        response()->json($user);
+    }
+
+    public function menteeInfo()
+    {
+        if (auth()->user()) {
+            $id = auth()->id();
+
+            $menteeInfo = db()
+            ->select("users
+            LEFT JOIN users_info
+            ON users.id = users_info.users_id
+            WHERE id = ?")
+            ->hidden("password")
+            ->bind($id)
+            ->fetchAssoc();
+
+            response()->json([
+                "menteeinfo" => $menteeInfo,
+            ]);
```

Comment:
do we need an array here as we return only one value?

Reply:
Check what the array returns, you will know if it returns only one element. This just returns all information about the user.

Reply:
If there's only one element then no need to wrap it in an array, we could do just
```php
response()->json($menteeInfo);
```

Am I right? 🤔

Reply:
Did you check what I asked in my previous comment?

Reply:
> Check what the array returns, you will know if it returns only one element. This just returns all information about the user.

You mean this one or another? Place a link if it's another. I'll check the return type soon but I'm almost sure it'll work